java/source/schema.rst

```diff
@@ -0,0 +1,330 @@
+===================
+Working with schema
+===================
+
+Common definition of table has an schema. Java arrow is columnar oriented and it also has an schema representation. 
+Consider that each name on the schema maps to a columns for a predefined data type
+
+
+.. contents::
+
+We are going to use this util for creating arrow objects:
```

Comment:
We should avoid as much as possible utility functions in the recipes, users should be able to copy/paste the recipe, only change the values to match with their own variables/constants and have it work. So requiring users to lookup for utilities that they need to embed in their code goes against the principle of the Cookbook.

If those utilities really add value, they should just be implemented in Arrow itself so that the cookbook and the users can rely on them.

Reply:
Updated to consider run every recipe independently without needed to run other code before